src/main/java/com/thirdcc/webapp/service/impl/EventAttendeeServiceImpl.java

```diff
@@ -42,9 +51,23 @@ public EventAttendeeServiceImpl(EventAttendeeRepository eventAttendeeRepository,
     @Override
     public EventAttendeeDTO save(EventAttendeeDTO eventAttendeeDTO) {
         log.debug("Request to save EventAttendee : {}", eventAttendeeDTO);
-        EventAttendee eventAttendee = eventAttendeeMapper.toEntity(eventAttendeeDTO);
-        eventAttendee = eventAttendeeRepository.save(eventAttendee);
-        return eventAttendeeMapper.toDto(eventAttendee);
+
+        Set<EventStatus> eventStatuses = new HashSet<EventStatus>() {{
+            add(EventStatus.OPEN);
+            add(EventStatus.POSTPONED);
+        }};
+
+        Event event = eventRepository
+            .findOneByIdAndStatusIn(eventAttendeeDTO.getEventId(), eventStatuses)
+            .get();
+
```

Comment:
Should check the event is not started also
something like 
`startDate < Instant.now()`

Reply:
Issue solved in 8b0a97596acf3f3e71e9c501c2b5f80cb66f3d97